src/cartridges/int_adyen_SFRA/cartridge/adyen/scripts/payments/getCheckoutPaymentMethods.js

```diff
@@ -39,6 +39,15 @@ const getExpressPaymentMethods = (expressPaymentMethods) => {
     );
 };
 
+const supportedStoredPaymentMethods = (storedPaymentMethods) =>
```

Comment:
Are we sure this properly filters, as I can't see the stored payment methods in the one-click use case?

Reply:
It should work fine now. I have changed the hardcode 'Ecommerce' to use value from constants.